en/micro-integrator/docs/references/mediators/property-Mediator.md

```diff
@@ -77,9 +77,28 @@ by using an XPath function. You can use any of the <a href="../../mediators/prop
 <li><strong>LONG</strong></li>
 <li><strong>SHORT</strong></li>
 <li><p><strong>OM</strong></p></li>
+<li><p><strong>JSON</strong></p></li>
 </ul>
 <p><strong>String</strong> is the default type.</p>
 <p>The <code>OM</code> type is used to set xml property values on the message context. This is useful when the expression associated with the property mediator evaluates to an XML node during mediation. When the <code>OM</code> type is used, the XML is converted to an AXIOM OMElement before it is assigned to a property.</p>
+<p>The <code>JSON</code> type is used to set JSON values on the message context. This comes handy when we use
```

Comment:
Can we explain what this means?
"This comes in handy when we use Property mediators with JSON payloads as a sub configuration in other native JSON supported mediators (for example, Enrich mediator)."

'... as a sub configuration in other native JSON-supported mediators'?

Reply:
This means we are using the property mediator inside another mediator configuration. Ex in Enrich we can use property as source or target configuration.
<enrich>
    <source type="property" property="NewSubject"/>	    
    <target action="child"  xpath="json-eval(data.students[1].modules)"/>	
</enrich>